go.mod

```diff
@@ -25,12 +25,13 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/stretchr/objx v0.3.0 // indirect
 	github.com/tdakkota/asciicheck v0.0.0-20200416200610-e657995f937b // indirect
+	github.com/terraform-providers/terraform-provider-vmc v1.3.0
```

Comment:
why are we still referring to the hashicorp's repo?

Reply:
Good catch. I will change it. I will also change reference to hashicorp's repo elsewhere in the code.

Reply:
I will make this change in a separate PR since I need to even change the vendor folder and also references to Hashicorp repo else where in the code like the Changelog.md, README.md etc